xml/art_migration.xml

```diff
@@ -0,0 +1,651 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<article version="5.1" xml:lang="en" xml:id="art.hos.migration"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Migration Guide</title>
+  <date><?dbtimestamp format="B d, Y"?></date>
+  <!-- <xi:include href="common_copyright_gfdl.xml"/> -->
+  <xi:include href="authors.xml"/>
+ </info>
```

Comment:
Some information about the target audience would be helpful: who should read this?